eng/Versions.props

```diff
@@ -40,8 +40,8 @@
   </PropertyGroup>
   <PropertyGroup Label="EmscriptenWorkloads">
     <!-- Workloads from dotnet/sdk -->
-    <MicrosoftNETWorkloadEmscriptenCurrentManifest100100TransportPackageVersion>10.0.100-rtm.25523.112</MicrosoftNETWorkloadEmscriptenCurrentManifest100100TransportPackageVersion>
-    <EmscriptenWorkloadManifestVersion>$(MicrosoftNETWorkloadEmscriptenCurrentManifest100100TransportPackageVersion)</EmscriptenWorkloadManifestVersion>
+    <MicrosoftNETWorkloadEmscriptenCurrentManifest100100PackageVersion>10.0.100</MicrosoftNETWorkloadEmscriptenCurrentManifest100100PackageVersion>
```

Comment:
Unrelated to your change, but what's the meaning of 100100 in this property name 🤔

Reply:
Maestro/darc are able to manage the versions by mapping the ID onto a property.

If you see in version.details.xml: Microsoft.NET.Workload.Emscripten.Current.Manifest-10.0.100
Remove all of the symbols not supported by MSBuild and add _PackageVersion_ to the end.

It's confusing as all the dots are gone so 10.0.100 becomes 100100.